en/flight_controller/intel_aero.md

```diff
@@ -93,3 +93,16 @@ The pinout for the Lidar Lite V3 is as follows
 ![](../../assets/hardware/Aero_I2C_splitter.JPG)
 
 ![](../../assets/hardware/Aero_LidarLite.JPG)
+
+## Using Optical Flow on the Aero
+
+The Intel Aero comes with a preinstalled optical flow binary (OS version 1.6 or higher) which enables the Aero to stably fly based on vision. In order to use optical flow, a range sensor has to be installed first (see above).
```

Comment:
- which should be "that"
- the use of "vision" seems imprecise. What does this actually provide - complete replacement to GPS?, velocity from optical flow.?